web/app/scripts/views/map/map-controller.js

```diff
@@ -88,6 +90,11 @@
                 params.polygon = userDrawnPoly;
             }
 
+            // Get a tilekey for the CSV export
+            QueryBuilder.djangoQuery(true, 0, _.extend({ limit: 0 }, params)).then(function(records) {
```

Comment:
It looks like this is causing a duplicate request to `records/?tilekey=true` every time the map filters change. It might be better for this to live in the export modal controller so that it only asks for a tilekey when the export button is clicked. Alternatively (and probably better), we could break out the tilekey functionality into a service so that there is only ever one tilekey request for each filter change and then the rest of the app can just pull from the service. That'd probably be a substantial refactor, however.